app/scripts/models/user.js

```diff
@@ -15,6 +15,7 @@ define(function (require, exports, module) {
   var Account = require('models/account');
   var Backbone = require('backbone');
   var Cocktail = require('cocktail');
+  var FlowModel = require('models/flow');
```

Comment:
Naming nit: read out of context, it's not obvious what a "Flow Model" would be for; I wonder if calling it something more verbose like "MetricsFlowModel" or "EventFlowModel" would make it more obvious what it's up to here.